.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -1,16 +1,18 @@
 ---
 name: Feature request
-about: Suggest an idea for Azeroth Pilot Reloaded
+about: Suggest an idea for Azeroth Pilot Reloaded.
 title: "[FEATURE] - "
-labels: "Feature"
+labels: Feature
```

Comment:
why did you delete the "" they are not mandatory?

Reply:
i never did lol. github did it itself

Reply:
so it's not useful when there is only one I think